nexus_constructor/add_component_window.py

```diff
@@ -532,18 +543,31 @@ def edit_existing_component(
         :return: The geometry object.
         """
         # remove the previous object from the qt3d view
-        self.parent().sceneWidget.delete_component(self.component_to_edit.name)
-
+        children_copy = deepcopy(self.component_to_edit.children)
+        for child in children_copy:
+            child.parent_node = None
```

Comment:
Why `parent_node` is set to `None` here?

Reply:
You are correct, it is not needed. I think that is some leftover from when I was debugging segmentation faults (tried everything :D ).